__init__.py

```diff
@@ -66,6 +66,9 @@ def __init__(self):
 
     def initialize(self):
         """Do these things after the skill is loaded."""
+        self.bus.on("homescreen.weather.update.request",
```

Comment:
could we make this event more generic? anything connected to the bus could request a weather report not only the homescreen

maybe use the namespace `f"{self.skill_id}.weather.request/response"`